class/Shadowsocks; Companion.json

```diff
@@ -230,10 +230,10 @@
 								"type": "options",
 								"entries": [
 									"On reaching 3rd level, you must dedicate yourself to one particular lifestyle that suits your role and your motivations: that of the Dervish, the Halcyon, the Psychopomp, or the Soothsayer, each detailed toward the end of the class description. Your choice grants features at the following levels: 3, 7, 11, and 15."
-								],
-								"gainSubclassFeature": true
+								]
 							}
-						]
+						],
+                                                “gainSubclassFeature”:true

```

Comment:
Those... aren't regular double quotes. What're you using to edit this file?

Reply:
Sorry, I was in a rush and tried editing a minor error directly on GitHub from mobile. I guess my phone doesn’t like to be normal. I can go back in and fix it once I’m back on my computer.